script/core.hbs

```diff
@@ -7,6 +7,14 @@
 <script src="https://assets.sitescdn.net/answers/v{{global_config.sdkVersion}}/{{#if params.sdkLocaleOverride}}{{#unless (eq params.sdkLocaleOverride 'en')}}{{params.sdkLocaleOverride}}-{{/unless}}{{else if global_config.locale}}{{#unless (eq global_config.locale 'en')}}{{global_config.locale}}-{{/unless}}{{/if}}answerstemplates.compiled.min.js" defer></script>
 <script>
 {{#babel}}
+  function onAnswersScriptLoad () {
+    const waitBeforeInitPromises = [
+      new Promise(HitchhikerJS.pageSpecificScriptsLoaded)
+    ];
+    Promise.all(waitBeforeInitPromises).then(() => {
```

Comment:
I'm using Promsie.all because this could be helpful for the dynamic config/JWT stuff. But if it's premature optimization then I can change this to just wait for the single promise.